### PR TITLE
Sabotaged MapEatCookie/GhostAttack/PacManConsume

### PR DESCRIPTION
**Pull request by SR02019**

### CODE REVIEW

File: Projects/P1/src/Ghost.java [hunk 1]
```diff
@@ -113,7 +113,7 @@ public boolean is_pacman_in_range() {
 	}
 
 	public boolean attack() {
-		if (is_pacman_in_range()) {
+		if (true) {
```
Comment (bhegde8) [inline Projects/P1/src/Ghost.java:116]: Your TestAttack will still pass despite this change. Maybe make it if (false) instead?

**Review by bhegde8 — CHANGES_REQUESTED** [review]

**Comment by SR02019** [comment]
Even though I changed my Ghost attackt to always return true, it still failed the test when I modified it. Which is why I kept it as "if (true)".

**Comment by xueweiji** [comment]
I test locally, it looks you are right.

**Review by xueweiji — COMMENTED** [review]
looks good.

**Review by xueweiji — COMMENTED** [review]
approve.

**Pull request merged**